torchvision/transforms/functional.py

```diff
@@ -68,7 +68,7 @@ def get_image_size(img: Tensor) -> List[int]:
     Returns:
         List[int]: The image size.
     """
-    _log_api_usage_once("torchvision.transforms.functional.get_image_size")
+    _log_api_usage_once("transforms", "get_image_size")
```

Comment:
@kazhang If I understand correctly, this will record the method as `torchvision.transforms.get_image_size`. Unfortunately this method doesn't actually exist because the `torchvision.transforms.functional` methods are not imported directly to `torchvision.transforms`.

Your previous modifications made sense because the recorded methods actually existed. For example, the call of `torchvision.ops.nms` is valid because that's an alternative path to access the method despite being defined under `torchvision.ops.boxes.nms`.

I'm afraid this can lead to confusion. @NicolasHug any thoughts on this?

Reply:
yes, if we can, it'd be preferable to follow the exact path